src/components/dropdown.ts

```diff
@@ -0,0 +1,27 @@
+import {Component, View} from 'angular2/angular2';
+
+@Component({
+  selector: 'uib-dropdown',
```

Comment:
@Foxandxss was against the `ng-` suggestion - we should probably hash this out in the meeting shortly. I would prefer `ng-bs-` myself.